bears/rest/RSTcheckBear.py

```diff
@@ -32,16 +33,32 @@ class RSTcheckBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         code_block_language_ignore: list=()):
+                         code_block_language_ignore: typed_list(str)=(),
```

Comment:
could you make a new commit out of this change? It's good to use `typed_list(str)` here, however it's a different issue :)

Reply:
done.